WORKSPACE

```diff
@@ -48,6 +48,10 @@ rules_haskell_toolchains(
     version = "8.8.3",
 )
 
+# Enable GHC persistent worker mode, if that's your bag.
+load("@rules_haskell//tools:repositories.bzl", "rules_haskell_worker_dependencies")
+
+rules_haskell_worker_dependencies()
```

Comment:
This block allows Bazel to run a GHC instance in the background and delegate build requests to it. This makes a lot of IDE’s more responsive. You can enable it by putting this in your `.bazelrc.local`:

```
build --worker_sandboxing=yes
build --define use_worker=True
```